modules/Model.js

```diff
@@ -1,23 +1,69 @@
 (function(root, factory) {
   if (typeof define === 'function' && define.amd) {
-    define(['underscore', 'backbone', './pollingMixin'], factory);
+    define(['underscore', 'backbone', './pollingMixin', './Events'], factory);
   } else if (typeof exports === 'object') {
-    module.exports = factory(require('underscore'), require('backbone'), require('./pollingMixin'));
+    module.exports = factory(require('underscore'), require('backbone'), require('./pollingMixin'), require('./Events'));
   } else {
     root.Torso = root.Torso || {};
-    root.Torso.Model = factory(root._, root.Backbone, root.Torso.Mixins.polling);
+    root.Torso.Model = factory(root._, root.Backbone, root.Torso.Mixins.polling, root.Torso.Events);
   }
-}(this, function(_, Backbone, pollingMixin) {
+}(this, function(_, Backbone, pollingMixin, Events) {
```

Comment:
We spoke at last night's Torso meeting about this patch. The direction you've taken this patch is great. One big thing that came up is that Torso.Events isn't quite the right place to put the timing events. Chat with @arikwex and @mandragorn about potential alternatives.